src/transformState.js

```diff
@@ -6,6 +6,34 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (let n = 0; n < actions.length; n++) {
+    switch (actions[n].type) {
+      case 'addProperties':
+        const actEntries = Object.entries(actions[n].extraData);
+
+        for (const [key, value] of actEntries) {
```

Comment:
Again, use full names in variables or just do this
```suggestion
        for (const [key, value] of Object.entries(actions[n].extraData)) {
```